ci/cpu/asan_ubsan_lsan.yml

```diff
@@ -14,7 +14,7 @@ cpu asan ubsan lsan deps:
   extends: .build_deps_common
   variables:
     EXTRA_APTGET: "clang-18 libclang-rt-18-dev libomp-18-dev llvm-18"
-    COMPILER: clang@18
+    COMPILER: llvm@18
```

Comment:
FYI, `clang` should still keep working if you prefer that instead because the alias defined here: https://github.com/spack/spack/blob/60f6f8d836b7cf31a82f49a58aa68123ef3af856/lib/spack/spack/aliases.py#L15.